functorch/functorch/_src/eager_transforms.py

```diff
@@ -260,13 +260,23 @@ def vjp(func: Callable, *primals, has_aux: bool = False):
         outer one. This is because ``vjp`` is a "function transform": its result
         should not depend on the result of a context manager outside of ``f``.
     """
+    return _vjp_with_argnums(func, *primals, has_aux=has_aux)
+
+
+# only to be used with jacrev to allow for all passed variables to be wrapped correctly, even if they don't participate
+# in autograd
```

Comment:
technically this is used in vjp and jacrev

Reply:
Should also note somewhere in the comment that the error messages are written from the perspective of the `vjp` API (instead of jacrev or something more neutral)

Reply:
Fixed comment to clarify that this shouldn't be user facing and is only split from vjp because of jacrev

Added note on the error messages